packages/forge2d/lib/src/dynamics/world.dart

```diff
@@ -750,7 +750,7 @@ class World {
           if (bodyA.sweep.alpha0 < bodyB.sweep.alpha0) {
             alpha0 = bodyB.sweep.alpha0;
             bodyA.sweep.advance(alpha0);
-          } else if (bodyB.sweep.alpha0 < bodyA.sweep.alpha0) {
+          } else {
```

Comment:
Not sure if it makes a difference, but this is not equivalent, what if they are the same?

Reply:
That's what I was thinking and that's why I set it to draft. Seems this could be an issue from the analyser, an issue was opened in 2017 regarding [this](https://github.com/dart-lang/linter/issues/811).

I'm thinking of rephrasing the logic and adding tests to see if I can get the analyser warning not showing up. If not it might be better to include an ignore rule comment.